# Stefan Teleman <stefan.teleman@oracle.com>
# We want MMX, SSE and SSE2 in libgomp.
# Internal patch. Submitted upstream. Status pending.
--- libgomp/testsuite/lib/libgomp.exp	2013-11-07 10:07:58.000000000 -0500
+++ libgomp/testsuite/lib/libgomp.exp	2014-11-19 08:37:47.706302934 -0500
@@ -145,7 +145,7 @@
     # We use atomic operations in the testcases to validate results.
     if { ([istarget i?86-*-*] || [istarget x86_64-*-*])
 	 && [check_effective_target_ia32] } {
-	lappend ALWAYS_CFLAGS "additional_flags=-march=i486"
+	lappend ALWAYS_CFLAGS "additional_flags=-march=pentium4"
     }
 
     if [istarget *-*-darwin*] {
